server/methods/accounts/accounts.js

```diff
@@ -741,16 +743,7 @@ export function sendWelcomeEmail(shopId, userId) {
 
   const user = Accounts.findOne(userId);
   const shop = Shops.findOne(shopId);
-
-  // Get shop logo, if available. If not, use default logo from file-system
-  let emailLogo;
-  if (Array.isArray(shop.brandAssets)) {
-    const brandAsset = _.find(shop.brandAssets, (asset) => asset.type === "navbarBrandImage");
```

Comment:
started to replace this `_.find` and realized there is a method which already does this